jarbas/dashboard/admin.py

```diff
@@ -85,7 +85,7 @@ class SuspiciousListFilter(JarbasListFilter):
     )
 
     def queryset(self, request, queryset):
-        return queryset.suspicions() if self.value() == 'yes' else queryset
+        return queryset.suspicions(self.value() == 'yes') if self.value() else queryset
```

Comment:
I think here we have a three state choice, don't we? Specially now we're paying attention to the so far unknown bug…

* If the filter is `yes` users should see only reimbursements **with suspicions**, i.e. `queryset.suspicions(True)`
* If the filter is `no`users should see only reimbursements **withot suspicions**, i.e. `queryset.suspicions(False)`
* If filter is not selected, the users see everything, i.e. `queryset`

Does that make sense?

Reply:
Yes, I see that way.
Do you think that organize the code like that could help to understand?

Reply:
Sure thing… I'd draft something among these lines:

```python
def queryset(self, request, queryset):
    filter = {
        'yes': queryset.suspicions(True),
        'no': queryset.suspicions(False)
    }
    return filter.get(self.value(), queryset)
```

Reply:
This is a nice way to handle the three choices, thanks @cuducos.

Reply:
Just beware that using `filter` is not recommended (because it overrides the native `filter` function). Choose anything else, even `filter_` instead ; )